README.md

```diff
@@ -0,0 +1,22 @@
+# Additional Fruits
+This module adds various different fruit blocks and includes 3 different stages for each plant.
```

Comment:
Beyond being a repository for various fruits this is a specific sort of content module for holding extra fruits that are part of the SimpleFarming system (see the `module.txt`) so it may help to dig around a little more to understand what connects to what :-)

Also "3 different stages" is overly verbatim - some fruits could easily use a different number of stages and it doesn't really help the user to know the specific number, just knowing that there are stages is enough